chatgpt-plugin/src/pages/api/metaplex/metaplex.yaml

```diff
@@ -0,0 +1,171 @@
+        Shows the public tree addresses with available space.
+      operationId: get_public_tree
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/getPublicTreeRequest"
+      responses:
+        "200":
+          description: Successful Response
+  /api/handlers/solana-pay/qr/createMintNFT:
+    post:
+      summary: createMintNFT
+      description: >-
+        Mints a normal uncompressed NFT with the given name, metadata uri and seller fees.
+      operationId: create_mint_nft
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/createMintNFTRequest"
+      responses:
+        "200":
+          description: Successful Response
+  /api/handlers/solana-pay/qr/createTransferAsset:
+    post:
+      summary: createTransferAsset
+      description: >-
+        Allows the user to send (transfer) compressed NFT to another account
+      operationId: create_transfer_asset
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/createTransferAssetRequest"
+      responses:
+        "200":
+          description: Successful Response
+  /api/handlers/solana-pay/qr/createBurnAsset:
+    post:
+      summary: createBurnAsset
+      description: >-
+        Allows the user to burn a compressed NFT
+      operationId: create_burn_asset
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/createBurnAssetRequest"
+      responses:
+        "200":
+          description: Successful Response
+components:
+  schemas:
+    getCNFTRentRequest:
+      title: GetCNFTRentRequest
+      type: object
+      required:
+        - treeSize
+        - canopySize
+      properties:
+        treeSize:
+          type: string
+          enum:
+            - "custom"
+            - "small"
+            - "medium"
+            - "large"
+        canopySize:
+          type: string
+          enum:
+            - "custom"
+            - "small"
+            - "medium"
+            - "large"
+            - "none"
+        customMaxDepth:
+          type: number
+          nullable: true
+        customMaxBufferSize:
+          type: number
+          nullable: true
+        customCanopySize:
+          type: number
+          nullable: true
+    createMintCNFTRequest:
+      title: CreateMintCNFTRequest
+      type: object
+      required:
+        - metadataUri
+      properties:
+        metadataUri:
+          type: "string"
+          format: "uri"
+          maxLength: 255
+    createMintNFTRequest:
+      title: CreateMintNFTRequest
+      type: object
+      required:
+        - name
+        - metadataUri
+      properties:
+        name:
+          type: string
+        metadataUri:
+          type: string
+          format: "uri"
+          maxLength: 255
```

Comment:
i don't think `format` and `maxLength` are respected by https://platform.openai.com/docs/guides/gpt/function-calling , but it's fine to leave for now